x/consensus/keeper/consensus/errors.go

```diff
@@ -14,3 +16,12 @@ const (
 
 	ErrAttestorNotSetForMessage = whoops.Errorf("attestator must be set for message: %T")
 )
+
+var (
+	ErrNilTypeCheck             = errors.New("TypeCheck can't be nil")
+	ErrNilBytesToSignCalculator = errors.New("BytesToSignCalculator can't be nil")
+	ErrNilVerifySignature       = errors.New("VerifySignature can't be nil")
+	ErrEmptyChainType           = errors.New("chain type can't be empty")
+	ErrEmptyChainReferenceID    = errors.New("chain id can't be empty")
+	ErrEmptyQueueTypeName       = errors.New("queue type name can't be empty")
+)
```

Comment:
FYI, if you want to keep `const` when declaring errors, you can take a look at `liberr` that basically wraps a `string` into an error, allowing you to define it as const. To me this is just flavour though, so up to you.